src/shogun/mathematics/zbsubs.h

```diff
@@ -0,0 +1,110 @@
+
```

Comment:
@karlnapf I am not sure what license to put here? This is mostly copied with some slight modifications to get it to compile without any additional libraries

Reply:
there was no license in the repo where I copied this from (probably because it is just C code translated directly from FORTRAN)

Reply:
It is probably public domain then. I suggest you try a bit to track the license down and if you cannot find anything State the source and that you modified it and say that it’s taken under public domain

Reply:
it seems like they went for GPL... http://r-forge.r-project.org/projects/specfun/
I can just do the f2c myself for the zbsubs routines?

Reply:
Gpl is non copy pastable
You can do them yourself but not sure how much work that would be?

Reply:
I translated it with f2c, it was some effort, but it should be working fine now

Reply:
nice one!